src/modules/completed.js

```diff
@@ -1,9 +1,12 @@
+/* eslint-disable import/no-cycle */
```

Comment:
Linters help us to write clean code so disabling them means we are doing ourselves a disservice. 

- [ ] Please kindly enable them and fix them instead.